### PR TITLE
[airspeed] reduce number of airspeed raw messages: 

### PR DESCRIPTION
**Pull request by dewagter**
ETS, MS45xx, SDP3x, UAVCAN -> RAW common message

 - [x] requires https://github.com/paparazzi/pprzlink/pull/176
 - [x] convert all existing sensors to output raw
 - [x] ETS hard sending -> register_telemetry
 - [x] add AIRSPEED_RAW to telemetry files

### CODE REVIEW

**Comment by EwoudSmeur** [comment]
This is great! Would be good to also place them in a RunOnceEvery macro to avoid flooding the datalink

**Comment by dewagter** [comment]
```make test``` and ```make test_coverty``` return

```
All tests successful.
Files=1, Tests=48, 466 wallclock secs ( 0.03 usr  0.00 sys + 2046.28 cusr 289.68 csys = 2335.99 CPU)
Result: PASS
```

File: sw/airborne/modules/sensors/airspeed_ets.c [hunk 1]
```diff
@@ -234,7 +234,11 @@ void airspeed_ets_read_event(void)
     stateSetAirspeed_f(airspeed_ets);
 #endif
 #if AIRSPEED_ETS_SYNC_SEND
-    DOWNLINK_SEND_AIRSPEED_ETS(DefaultChannel, DefaultDevice, &airspeed_ets_raw, &airspeed_ets_offset, &airspeed_ets);
+    uint8_t dev_id = AIRSPEED_ETS_ID;
+    float press = 0;
+    float temp = 0;
+    float offset = airspeed_ets_offset;
+    DOWNLINK_SEND_AIRSPEED_RAW(DefaultChannel, DefaultDevice, &dev_id, &airspeed_ets_raw, &offset, &press, &temp, &airspeed_ets);
```
Comment (dewagter) [inline sw/airborne/modules/sensors/airspeed_ets.c:241]: - change to register periodic
 - check telemetry.xml

**Comment by dewagter** [comment]
> This is great! Would be good to also place them in a RunOnceEvery macro to avoid flooding the datalink

Does ```register_periodic_telemetry``` not use the frequency from the telemetry.xml?

**Comment by fvantienen** [comment]
Both the modules and coverity are successful

**Review by fvantienen — APPROVED** [review]

**Comment by EwoudSmeur** [comment]
> > This is great! Would be good to also place them in a RunOnceEvery macro to avoid flooding the datalink
> 
> Does `register_periodic_telemetry` not use the frequency from the telemetry.xml?

Of course, you already changed it from sending a direct downlink message to registering it! Sorry I missed that, it's perfect!

**Pull request merged**